ExpensesTracker/Forms/MainWindow.cs

```diff
@@ -139,27 +154,34 @@ private void SavePdfButton_Click(object sender, EventArgs e)
 
         private void RefreshAccountsData()
         {
+            accountBox.SelectedIndexChanged -= (SelectedAccountBox_SelectedIndexChanged);
```

Comment:
Is it required here? 
Just asking. If it's required then keep it.

Reply:
yes, its required